### ISSUE TITLE
[Accessibility]: Code Insights Dashboards - Shift + Tab jumps to last card

### ISSUE THREAD

**Issue by unclejustin** [sourcegraph/sourcegraph#41690]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33492

### Problem description

https://www.loom.com/share/4ce184a38fa643659924b4725f9ffa86

On the Code Insights dashboard page, tab until the Dashboard dropdown is focused. Then down arrow to see the list. Down arrow again to select another dashboard then shift + tab. It jumps to the last insight in the list.

### Expected behavior

Should focus back to the tab panel group.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

/cc @joelkw @felixfbecker @vovakulikov

**Comment by vovakulikov** [comment]
This problem was closed with dropdown select UI migration to the new combobox UI implementation.
